src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Microsoft.AspNetCore.Razor.LanguageServer.Test.csproj

```diff
@@ -13,6 +13,19 @@
     <None Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Microsoft.WebTools.Languages.LanguageServer.Server" HintPath="C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\IDE\Extensions\Microsoft\Web Tools\Languages\Microsoft.WebTools.Languages.LanguageServer.Server.dll" />
+    <Reference Include="Microsoft.WebTools.Languages.Shared" HintPath="C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\IDE\Extensions\Microsoft\Web Tools\Languages\Microsoft.WebTools.Languages.Shared.dll" />
+    <Reference Include="Microsoft.WebTools.Languages.Shared.Editor" HintPath="C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\IDE\Extensions\Microsoft\Web Tools\Languages\Microsoft.WebTools.Languages.Shared.Editor.dll" />
+    <Reference Include="Microsoft.WebTools.Shared" HintPath="C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\IDE\Extensions\Microsoft\Web Tools Shared\Microsoft.WebTools.Shared.dll" />
+    <Reference Include="Microsoft.VisualStudio.Web" HintPath="C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\IDE\Microsoft.VisualStudio.Web.dll" />
+  </ItemGroup>
```

Comment:
@jimmylewis These are the dependencies I need for the tests to work

Reply:
Quick update: MS.WebTools.Shared and MS.VS.Web are now on the vs-impl feed.  I'm hoping to have the rest there early next week (pending PR, build, and publishing).

Reply:
These should all be published now.  Let me know if there is any issue with them, or any other dependencies missing.

Reply:
Thanks, I'll try it out today!